ex9/js/main.js

```diff
@@ -0,0 +1,95 @@
+require(['ajaxGet'], function(ajaxGet) {
+  $(function() {
+    var on;
+    setEvents();
+    on = {
+      loadList: function() {
+         ajaxGet('json/data.json', function(phoneBookData) {
+            $('li').remove();
+            $("#phoneBookTemplate").tmpl( phoneBookData )
+              .appendTo( "#phoneBook" );
+          });
+      },
+      searchInput: function() {
+        console.log('searchInput');
+        if($('#textSearch').val() === '') {
+          $( "#btnFilter" ).prop( "disabled", true );
+        } else {
+          $( "#btnFilter" ).prop( "disabled", false );
+        }
+      },
+      searchChange: function() {
+        console.log('searchChange');
```

Comment:
This repeats exactly what's above. It normally implies you need to refactor the code so there's no repetition.